addons/rose/package.json

```diff
@@ -66,11 +66,12 @@
     "@storybook/cli": "^6.3.10",
     "@storybook/components": "^6.3.12",
     "@storybook/core-events": "^6.3.0",
-    "@storybook/ember": "^6.3.8",
+    "@storybook/ember": "6.4.0",
     "@storybook/ember-cli-storybook": "^0.6.0",
     "@storybook/theming": "^6.3.0",
     "babel-loader": "^8.2.5",
     "babel-plugin-ember-modules-api-polyfill": "^2.12.0",
+    "babel-plugin-htmlbars-inline-precompile": "3",
```

Comment:
Why is this needed and how come it's pinned to `3`? It looks like it's an archived package

Reply:
It is a peerDep dependency, I know is pin to 3 and not ideal, but is how storybook 6.4 has it defined. Hopefully, soon we will upgrade Storybook and this could be gone.

I left a screenshot I took during development that shows the peer dep, look at the ones pending on `rose`

![Screenshot from 2023-06-15 15-37-06](https://github.com/hashicorp/boundary-ui/assets/9775006/1d2f8052-eb4b-4953-96d8-23e6401a2563)